lib/pdksync.rb

```diff
@@ -92,8 +101,13 @@ def self.main(steps: [:clone, :pdksync, :createpr], args: nil)
         print 'updated, '
         next unless pdk_update(@output_path).exitstatus.zero?
       end
-      if steps.include?(:create_pr)
-        print 'PR, '
+      if steps.include?(:create_commit)
+        print 'commit created, '
```

Comment:
I'd move this output to below the command execution incase it breaks, it'd be confusing for it to tell you the commit is created before it actually is.